site/src/sphinx/setup-configuration.rst

```diff
@@ -66,7 +67,12 @@ Core properties
 
   - ``protocol`` (string)
 
-    - the protocol. ``http`` is the only supported protocol at the moment.
+    - the protocol. ``http`` and ``https`` are supported.
+
+- ``tls``
+
+  - the configuration for Transport Layer Security(TLS) support. Specify ``null`` to disable TLS.
+    It will be explained later in this page.
```

Comment:
Could you use hyperlink? See `concepts.rst` and `client-java.rst` for an example: https://github.com/line/centraldogma/search?l=reStructuredText&q=concepts&type=&utf8=%E2%9C%93